examples/guided.py

```diff
@@ -22,34 +22,6 @@
 # For support reasons, we'll log the disk layout pre installation to match against post-installation layout
 archinstall.log(f"Disk states before installing: {archinstall.disk_layouts()}", level=logging.DEBUG)
 
-def load_config():
-	if archinstall.arguments.get('harddrives', None) is not None:
-		if type(archinstall.arguments['harddrives']) is str:
-			archinstall.arguments['harddrives'] = archinstall.arguments['harddrives'].split(',')
```

Comment:
Where did these logics go? Are they baked in some where else? :)

Reply:
Ah nvm, I just did `ctrl+f` and looked for it, but it's changed slightly to `arguments['harddrives'] = arguments['harddrives'].split(',')` heh. My bad!

Reply:
It did cost me a bit to understand what it was about. Probably some legacy config files to convert ... This lines are there since i started with the code